src/modules/UI/scenes/TransactionList/TransactionListConnector.js

```diff
@@ -56,6 +57,8 @@ const mapStateToProps = (state: State) => {
   const currencyInfo: EdgeCurrencyInfo = coreWallet.currencyInfo
   // set default requiredConfirmations to 1, so once the tx is in a block consider fully confirmed
   const requiredConfirmations = currencyInfo.requiredConfirmations ? currencyInfo.requiredConfirmations : 1
+  // eventually we will want the txList balance box visibility to be set to wallet-level setting (not account)
```

Comment:
comments add clarification to existing code
dont use them as a todo list

Reply:
Okay. I have added an Asana task:
https://app.asana.com/0/361770107085503/734196653255276